test/helm-test-e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+# Install and initialize helm/tiller
+HELM_URL=https://storage.googleapis.com/kubernetes-helm
+HELM_TARBALL=helm-v2.2.3-linux-amd64.tar.gz
```

Comment:
2.2.3 is needed at a minimum I think because it fixed some issues with `--wait`.